### PR TITLE
Prepare omero-marshal for 0.4.0 release

### PR DESCRIPTION
**Pull request by sbesson**
With the upcoming merge of #12, this should be the last PR required for tagging and uploading `omero-marshal 0.4.0` to Pypi.

- bump VERSION in setup.py
- add Changelog for the feature additions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.4.0 (2016-07-02)
```
Comment (mtbc) [inline CHANGELOG.md]: not 09-02?

Comment (sbesson) [inline CHANGELOG.md]: Correct, pushed

**Comment by will-moore** [comment]
Looks good!

**Comment by sbesson** [comment]
Merging for release

**Pull request merged**